Ecommerce/index.css

```diff
@@ -1 +1,78 @@
+        <link rel="stylesheet" href="style.css">
+        <style>
+            .top{
+                background-color:#4D133F;
+                width:100%;
+                height:11rem;
+                text-align:center;
+                color:#ffff;
+                font-size:0.8em;
+                margin-top:-1rem;
+            }
+            .box{
+              background-color:#E8E8E9;
+              width:100%;
+              height:100%;
+            }
+            .box-container{
+             display:grid;
+             background-color:#E8E8E9;
+             grid-template-columns:auto;
+             }
+             .box-item{
+             background-color:#ffff;
+             margin:0.5rem 1rem 0.5rem 1rem;
+             box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.2);
+             font-size:1em;
+            }
+           input{
+            width:100%;
+           border-color:#555;
+           border-radius:5px;
+           }
+          .b{
+         
+          box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.1);
+          margin:0rem 1rem 1rem 1rem;
+          }
+        </style>
+    </head>
+    <body>
+     <div class="top"><h2><br>Welcome to My Shop</h2>
+<img src="shopping_cart.png" style="width:150px;height:70px;">
+<h3>About Us&nbsp;&nbsp;&nbsp;Products&nbsp;&nbsp;&nbsp;Contact Us</h3></div>
+<div class="box">
+<div class="box-container">
+<div class="box-item"><h2 style="color:#4D133F;margin-left:1rem;">About Us</h2>
+<p style="margin-left:1rem;text-align:justify;margin-right:1rem;">Welcome to My Shop! We are a one -stop destination for all your shopping needs. Our mission is to provide you with the best products and exceptional customer service.</p>
+</div>
+<div class="box-item"><h2 style="color:#4D133F;margin-left:1rem;">Products</h2>
+<p style="margin-left:1rem;">Explore our wide range of products, including:</p>
+<ul style="margin-left:1rem;"><li>Electronics</li><li>Clothing</li><li>Home Decor</li><li>Toys</li></ul>
+<p style="margin-left:1rem;text-align:justify;margin-right:1rem;">Find perfect items for yourself, or as gift for your loved ones.</p>
+</div>
+<div class="box-item"><h2 style="color:#4D133F;margin-left:1rem;">Contact Us</h2>
+<p style="margin-left:1rem;text-align:justify;margin-right:1rem;">If you have any questions or need assistance, please don't hesitate to reach out to us using the contact form below:</p>
+<div class="b">
+<form action="#" style="margin-left:1rem;margin-right:1rem;">
+<label for="name"><b>Name:</b><br></label>
+<input type="text" id="name"  style="height:2rem;"><br><br>
+<label for="email"><b>Email:</b><br></label>
+<input type="text" id="email" style="height:2rem;"><br><br>
+<label for="message"><b>Message:</b><br></label>
+<input type="text" id="message" style="height:4rem;"><br><br>
+<button style="background-color:#4D133F;color:#ffff;border-radius:5px;width:4rem;height:2rem;margin-bottom:1rem;"> Submit</button>
+</div>
+

```

Comment:
There are two empty lines at the end of the code. It is recommended to remove them for better readability.